.env.test

```diff
@@ -1,6 +1,6 @@
 GOVUK_NOTIFY_API_KEY=override-locally
-CHECK_RECORDS_DOMAIN=http://127.0.0.1
-HOSTING_DOMAIN=http://127.0.0.1
+CHECK_RECORDS_DOMAIN=http://check_records.localhost
+HOSTING_DOMAIN=http://qualifications.localhost
```

Comment:
Minor, and can be addressed later, but if we drop the protocol here, we can use these env vars in the various configuration bits below (and simply interpolate a string where a protocol is needed, ie—the capybara app host).